src/pages/studyView/resources/FilesAndLinks.tsx

```diff
@@ -34,16 +34,21 @@ class FilesLinksTableComponent extends LazyMobXTable<{
 const RECORD_LIMIT = 500;
 
 function getResourceDataOfEntireStudy(studyIds: string[]) {
-    // Only handle the first studyId for now. Can be expanded to make a call per
-    // studyId.
-    const studyId = studyIds[0];
-    const allResources = internalClient.getAllStudyResourceDataInStudyPatientSampleUsingGET(
-        {
-            studyId: studyId,
-            projection: 'DETAILED',
-        }
+    // Fetch resource data for each studyId, then return combined results
+    const allResources = _(studyIds)
```

Comment:
the js array natively has map on it so you don't need to wrap in lodash here.  this might seem confusing given what i said last week but the real message is that we should strive to use the semantically accurate and descriptive method.  if it's just map, then no reason to use lodash.   the other reason to use lodash is it's support for chaining, but we're not doing that here.

Reply:
Okay no problem I can change to the native map. I was just going off of what the previous author in another function in this same file.